### DISCUSSION TITLE
Consider original Discord markings in Discordian Theme

### DISCUSSION THREAD

**Discussion by radekkozak** [radekkozak/discordian#10] [Ideas]
![unknown](https://user-images.githubusercontent.com/2744747/103925850-6aee5000-5118-11eb-958a-60849ec20e1f.png)

Let's consider markdown styles to be more like actual Discord styles. **Would it be a good choice as default or is it better to have it as user-controllable thing via Discord Plugin**  

The main question is how such settings in big block of texts would affect end-user working with it and how it'd benefit the overall visuals in regard to straining the eye

[![](https://api.gh-polls.com/poll/01EVF07FNPG8QTN9KFYNG0S98N/Current%20as%20default%20-%20Discord%20original%20as%20option)](https://api.gh-polls.com/poll/01EVF07FNPG8QTN9KFYNG0S98N/Current%20as%20default%20-%20Discord%20original%20as%20option/vote)
[![](https://api.gh-polls.com/poll/01EVF07FNPG8QTN9KFYNG0S98N/Discord%20original%20as%20default%20-%20Current%20as%20option)](https://api.gh-polls.com/poll/01EVF07FNPG8QTN9KFYNG0S98N/Discord%20original%20as%20default%20-%20Current%20as%20option/vote)
[![](https://api.gh-polls.com/poll/01EVF07FNPG8QTN9KFYNG0S98N/Leave%20it%20all%20for%20user%20-%20to%20set%20via%20CSS%20snippets)](https://api.gh-polls.com/poll/01EVF07FNPG8QTN9KFYNG0S98N/Leave%20it%20all%20for%20user%20-%20to%20set%20via%20CSS%20snippets/vote)

**Here's the current settings:**
 
![dark-mode-discordian](https://user-images.githubusercontent.com/2744747/103926283-fec01c00-5118-11eb-9313-4ec648fd9ec9.png)

**Here's what's more or less proposed** when we go for pure Discord choice:

![dark-mode-original](https://user-images.githubusercontent.com/2744747/103926280-fd8eef00-5118-11eb-8700-b4e0ae4d24d6.png)

**Comment by phibr0** [comment]
Would love that! Maybe optional in the Settings?

**Reply by radekkozak** [reply]
Hi Philip, will definitely take it under account as there are few voices about it already. Option in the plugin would probably be the wisest choice to satisfy both sides. It should made its way into the next release i hope. Thanks for the comment, appreciated

**Comment by radekkozak** [comment]
**This has just been added in 0.5.0 version** of Discordian Theme & Plugin /cc @phibr0
